[05-Mar-2025 09:05:35 Europe/Berlin] Attempting database connection to localhost with user root
[05-Mar-2025 09:05:35 Europe/Berlin] Database connection successful
[05-Mar-2025 09:05:35 Europe/Berlin] Starting trace for approval process with ID: 20
[05-Mar-2025 09:05:35 Europe/Berlin] Query execution error: Commands out of sync; you can't run this command now
[05-Mar-2025 09:05:35 Europe/Berlin] Query: UPDATE studentpendingenroll SET status = 'approved' WHERE id = ?
[05-Mar-2025 09:05:35 Europe/Berlin] Trace error: Commands out of sync; you can't run this command now
[05-Mar-2025 09:05:59 Europe/Berlin] Attempting database connection to localhost with user root
[05-Mar-2025 09:05:59 Europe/Berlin] Database connection successful
[05-Mar-2025 09:05:59 Europe/Berlin] Starting trace for approval process with ID: 20
[05-Mar-2025 09:05:59 Europe/Berlin] Query execution error: Commands out of sync; you can't run this command now
[05-Mar-2025 09:05:59 Europe/Berlin] Query: UPDATE studentpendingenroll SET status = 'approved' WHERE id = ?
[05-Mar-2025 09:05:59 Europe/Berlin] Trace error: Commands out of sync; you can't run this command now
[05-Mar-2025 09:06:00 Europe/Berlin] Attempting database connection to localhost with user root
[05-Mar-2025 09:06:00 Europe/Berlin] Database connection successful
[05-Mar-2025 09:06:00 Europe/Berlin] Starting trace for approval process with ID: 20
[05-Mar-2025 09:06:00 Europe/Berlin] Query execution error: Commands out of sync; you can't run this command now
[05-Mar-2025 09:06:00 Europe/Berlin] Query: UPDATE studentpendingenroll SET status = 'approved' WHERE id = ?
[05-Mar-2025 09:06:00 Europe/Berlin] Trace error: Commands out of sync; you can't run this command now
[05-Mar-2025 09:08:21 Europe/Berlin] Attempting database connection to localhost with user root
[05-Mar-2025 09:08:21 Europe/Berlin] Database connection successful
[05-Mar-2025 09:08:21 Europe/Berlin] Starting trace for ID: 20
[05-Mar-2025 09:08:21 Europe/Berlin] PHP Fatal error:  Uncaught ArgumentCountError: The number of variables must match the number of parameters in the prepared statement in C:\xamppNew\htdocs\CST5-PROJECT\FINAL-ADMIN\Admin-PHP\trace_query.php:42
Stack trace:
#0 C:\xamppNew\htdocs\CST5-PROJECT\FINAL-ADMIN\Admin-PHP\trace_query.php(42): mysqli_stmt->bind_param('s', 20)
#1 C:\xamppNew\htdocs\CST5-PROJECT\FINAL-ADMIN\Admin-PHP\trace_query.php(75): SQLTrace->executeQuery('SELECT * FROM s...', Array)
#2 {main}
  thrown in C:\xamppNew\htdocs\CST5-PROJECT\FINAL-ADMIN\Admin-PHP\trace_query.php on line 42
[05-Mar-2025 09:10:00 Europe/Berlin] Attempting database connection to localhost with user root
[05-Mar-2025 09:10:00 Europe/Berlin] Database connection successful
[05-Mar-2025 09:10:00 Europe/Berlin] Starting trace for ID: 20
